lib/types.ts

```diff
@@ -2620,6 +2635,15 @@ export type MembersCountResponse = {
   count: number;
 };
 
+export type GetRichMenuAliasResponse = {
+  richMenuAliasId: string;
+  richMenuId: string;
+};
+
+export type GetRichMenuAliasListResponse = {
+  aliases: GetRichMenuAliasResponse[];
+};
```

Comment:
nitpick: seems more gentle if would set comments!

```suggestion
/**
 * @see [Get rich menu alias information](https://developers.line.biz/en/reference/messaging-api/#get-rich-menu-alias-by-id)
 */
export type GetRichMenuAliasResponse = {
  richMenuAliasId: string;
  richMenuId: string;
};

/**
 * @see [Get list of rich menu alias](https://developers.line.biz/en/reference/messaging-api/#get-rich-menu-alias-list)
 */
export type GetRichMenuAliasListResponse = {
  aliases: GetRichMenuAliasResponse[];
};
```

Reply:
I forgot it! I'll do it in next time.